README.md

```
@@ -33,42 +33,61 @@ pmcp --prom-addr="http://localhost:9090" --transport=stdio
 
 ### Command Line Flags
 
-- `--prom-addr`: Prometheus server address (default: http://localhost:9090/)
-- `--mcp-addr`: MCP server listening address (default: localhost:8080)
-- `--transport`: Transport mechanism (stdio, http, sse) (default: stdio)
+- `-mcp-addr`: The address of the MCP server to listen on. (default "localhost:8080")
+- `-prom-addr`: The address of the Prometheus to connect to. (default "http://localhost:9090/")
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)